src/helpers.ts

```diff
@@ -361,17 +405,18 @@ module.exports = apiVersion => {
    *     console.error(err);
    *   });
    */
+
   methods.landmarkDetection = promisify(
-    _createSingleFeatureMethod(features.LANDMARK_DETECTION)
+    _createSingleFeatureMethod(features!.LANDMARK_DETECTION)
   );
 
   /**
-   * Annotate a single image with logo detection.
+   * Annotate a single image with text detection.
    *
    * @see v1.ImageAnnotatorClient#batchAnnotateImages
    * @see google.cloud.vision.v1.AnnotateImageRequest
    *
-   * @method v1.ImageAnnotatorClient#logoDetection
+   * @method v1.ImageAnnotatorClient#textDetection
```

Comment:
A little bit weird that the order of feature methods is changed. Can we double check that we haven't lost any of those?

Reply:
Yes, I have compared them, the order is only because me copy paste :) But I will reorder it to make it aligned with the original file, that would make more sense. Thanks!